### PR TITLE
2007 inside connector script fix

### PR DESCRIPTION
**Pull request by MaxTru**
__Which issue does this PR address?__

Closes [camunda/camunda-modeler#2007](https://github.com/camunda/camunda-modeler/issues/2007) (fix-upstream)

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: test/spec/provider/camunda/ConnectorSpec.js [hunk 1]
```diff
@@ -1083,7 +1082,7 @@ function getOutputParameterGroupLabel(container) {
 }
 
 function getParameterNameInput(container) {
-  return domQuery('input[id*="parameterName"]', getInputParametersGroup(container));
+  return domQuery('input[name="name"]', getInputParametersGroup(container));
```
Comment (pinussilvestrus) [inline test/spec/provider/camunda/ConnectorSpec.js:1085]: Why is this change needed?

File: test/spec/provider/camunda/InputOutputParameterTextSpec.js [hunk 2]
```diff
@@ -727,6 +816,10 @@ function getCursorPosition() {
   return range.startOffset;
 }
 
+function getParameterScriptValueInput(container, id) {
+  return domQuery('input[id="input-parameter-' + id + 'cam-script-resource-val"]', getInputParametersGroup(container));
```
Comment (pinussilvestrus) [inline test/spec/provider/camunda/InputOutputParameterTextSpec.js:820]: So it's basically not the "script value", but the "script resource value"? Maybe we should make it clear then, also [in the test itself](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/401/files#diff-3f0f09702496b6cf60a8e76b7a70db2b75efd7aa92c265f073e70a944e4c84d9R700), to not get confused by the other "script value", namely the Inline Script.

`getParameterScriptResourceValue`

**Review by pinussilvestrus — COMMENTED** [review]
Works well 👏   just two minor thoughts to consider.

File: test/spec/provider/camunda/ConnectorSpec.js (hunk 1 above)
Comment (MaxTru) [reply test/spec/provider/camunda/ConnectorSpec.js:1085]: There is no factual reason. I worked on this and played with this and accidentally rewrote it. But the result is the same. I will revert the change so to not cause confusion.

File: test/spec/provider/camunda/InputOutputParameterTextSpec.js (hunk 2 above)
Comment (MaxTru) [reply test/spec/provider/camunda/InputOutputParameterTextSpec.js:820]: True, will do

**Review by pinussilvestrus — APPROVED** [review]
